src/functions.c

```diff
@@ -819,7 +819,7 @@ void functionFlushCommand(client *c) {
 /* FUNCTION HELP */
 void functionHelpCommand(client *c) {
     const char *help[] = {
-"LOAD <ENGINE NAME> <LIBRARY NAME> [REPLACE] [DESCRIPTION <LIBRARY DESCRIPTION>] <LIBRARY CODE>",
+"LOAD [REPLACE] <FUNCTION CODE>",
```

Comment:
I would expect this to be:
```suggestion
"LOAD [REPLACE] <LIBRARY CODE>",
```
But it seems like that isn't consistent with the docs anyways.